packages/build/src/error/api.js

```diff
@@ -24,9 +25,24 @@ const apiMethodHandler = async function(endpoint, method, parameters, ...args) {
   try {
     return await method(parameters, ...args)
   } catch (error) {
+    redactError(error)
     addErrorInfo(error, { type: 'api', location: { endpoint, parameters } })
     throw error
   }
 }
 
+// Redact API token from the build logs
+const redactError = function(error) {
```

Comment:
This looks good.
Not sure if we want to redact `POST` data as well as it might contain sensitive information in the future (doesn't contain anything sensitive at the moment).
We could have a list of `POST` data properties that we allow to be printed to avoid future leaks if we start sending more information.

Reply:
Makes sense!
I created an issue at https://github.com/netlify/build/issues/1530